org.eclipse.draw2d.tests/src/org/eclipse/draw2d/test/TestImages.java

```diff
@@ -19,23 +19,16 @@
 
 public class TestImages {
 
-	static final Image depth_8;
-	static final Image depth_24;
+	static final Image depth_8 = loadImage("icons/bits8.jpg"); //$NON-NLS-1$
+	static final Image depth_24 = loadImage("icons/bits24.jpg"); //$NON-NLS-1$
 
-	static {
-		InputStream is = TestImages.class.getResourceAsStream("icons/bits8.gif"); //$NON-NLS-1$
-		depth_8 = new Image(null, is);
-		try {
-			is.close();
-		} catch (Exception e) {
-		}
+	private static Image loadImage(final String imageName) {
+		try (InputStream is = TestImages.class.getResourceAsStream(imageName)) { // $NON-NLS-1$
+			return new Image(null, is);
 
-		is = TestImages.class.getResourceAsStream("icons/bits24.jpg"); //$NON-NLS-1$
-		depth_24 = new Image(null, is);
-		try {
-			is.close();
 		} catch (Exception e) {
 		}
+		return null;
```

Comment:
This whole expression I would simplify to `ImageDescriptor.createFromFile(rsrcClass, name).createImage()` / `ImageDescriptor.createFromFile(TestImages.class, name).createImage()`.

Reply:
Thx for pointing this out. Changed!